README.md

```diff
@@ -23,13 +24,14 @@ Note on Windows: you have to set the correct path to `OGRETOOLS_XML_CONVERTER` i
 After installing the addon enable it in Blender from `User Preferences > Add-Ons > Import-Export` (CTRL+ALT+U). Search for `ogre` and check the box on the right. Remember to save as default if you want the addon to be enabled after you exit your Blender.  
 Integrated help docs will be shown in the upper right hand toolbar, replacing blender's normal `Help` menu, read them for assistance.
 
-The following versions of blender have been tested. It might also work with older versions of Blender.
+The following versions of blender have been tested.
 
-* Blender 2.7x
```

Comment:
did you want to target master with this?

Reply:
Thanks for catching that one, I forgot that the READMEs are not exactly the same between 2.7 and master